samples/javascript_typescript/60.multilingual-luis-bot/translator.ts

```diff
@@ -0,0 +1,345 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import * as request from 'request-promise-native';
+import { promises } from 'fs';
+
+
+export class TranslateArrayOptions{
+    texts: string[];    from: string;
+    to: string;
+    contentType?: string;
+    category?: string;
+}
+
+/**
+ * @private
+ */
+export class TranslationResult {
+    translatedText: string;
+}
+
+/**
+ * @private
+ */
+export class TranslationRequest{
+	Text: string;
+}
+
+/**
+ * @private
+ */
+export class MicrosoftTranslator {
+    
+    apiKey: string;
+    postProcessor: TranslatorPostProcessor;
+
+    constructor(apiKey: string, noTranslatePatterns?: {[id: string] : string[]}, wordDictionary?: { [id: string]: string }) {
```

Comment:
Do this 
```suggestion
    constructor(private apiKey: string, noTranslatePatterns?: {[id: string] : string[]}, wordDictionary?: { [id: string]: string }) {
```
and get rid of lines 34 and 38